tool/gravity/cli/status.go

```diff
@@ -384,6 +384,12 @@ func printNodeStatus(node statusapi.ClusterServer, w io.Writer) {
 	if node.Profile != "" {
 		description = fmt.Sprintf("%v / %v", description, node.Profile)
 	}
+	if node.InstanceID != "" {
+		description = fmt.Sprintf("%v / %v", description, node.InstanceID)
```

Comment:
This is going to overwrite the description with the last available attribute - i.e. with teleport instance type and suppress both the profile and cloud-specific instance ID - I think the idea was to include them all if available.
Also, I would use the descriptive header for the attributes so it was obvious which is which:
```
10.128.0.18 / node
Instance ID:   i-deadbeef
Instance Type: foo
```